app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java

```diff
@@ -44,7 +47,7 @@ public Fragment getItem(int position) {
             case 0:
                 return AccountFragment.newInstance(null);
             case 1:
-                return AccountFragment.newInstance(null);
+                return new RankingContainerFragment();
```

Comment:
I see we are using static in the others implementations. Why this one is just calling the constructor?